dpbench/benchmarks/l2_norm/l2_norm_numba_dpex_n.py

```diff
@@ -7,9 +7,7 @@
 
 
 @nb.njit(parallel=False, fastmath=True)
```

Comment:
The `parallel` flag should not be set to `False`. dpex always requires parallel to True.

Reply:
Switched true.